.github/workflows/test.yaml

```diff
@@ -16,4 +16,37 @@ jobs:
           go-version: 1.x
       - run: make check
       - run: make check-generate
-      - run: make check-envtest
+
+  kubernetes-api:
+    runs-on: ubuntu-latest
+    needs: [go-test]
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes: [default]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with: { go-version: 1.x }
+      - run: go mod download
+      - env: { KUBERNETES: "${{ matrix.kubernetes }}" }
+        run: ENVTEST_K8S_VERSION="${KUBERNETES#default}" make check-envtest
+
+  kubernetes-k3d:
+    if: "${{ github.repository == 'CrunchyData/postgres-operator' }}"
```

Comment:
I forget, why does this test (check-envtest-existing) require the `if` repo check but the other (check-envtest) doesn't :question:

Reply:
We opted to exclude this test from any private repos.